### ISSUE TITLE
Linter does not detect declared properties in inherited behaviors

### ISSUE THREAD

**Issue by dman777** [PolymerLabs/polylint#83]
```javascript
<template is="dom-if" if="{{show}}">
<script>
    Polymer({
      is: "resume-page",
      properties: {
        printButtonHidden: {
          type: Boolean,
          value: true
        }
      },
      observers: ['hidePrintButton(show)'],
      behaviors: [pageBehavior],
      hidePrintButton: function(show) {
        if (!show) {
          this.printButtonHidden = true;
        }
      },
``` 

`show` is declared in `pageBehavior`... but the linter does not catch this and displays:

` Property show not found in 'properties' for element 'resume-page'`

##### polylint@2.9.0

**Comment by saeidzebardast** [comment]
Same problem!
**polylint@2.9.0**

**Comment by mderriey** [comment]
You have to decorate your behavior with a `@polymerBehavior` tag for polylint to pick it up.

```js
/**
 * @polymerBehavior
 */
MyBehavior = {
    property: String
}
```

**Comment by saeidzebardast** [comment]
@mderriey Thank you. It fixed the warnings.

**Comment by PBM42** [comment]
How do you fix it when extending behaviors ?

**Comment by gertcuykens** [comment]
@arthurevans please mention this
```
/**
 * @polymerBehavior MyBehavior
 */
```
add https://www.polymer-project.org/1.0/docs/devguide/behaviors.html

**Comment by bendavis78** [comment]
Adding @polymerBehavior to my behavior did not fix the linter errors for me. I still get "property ____ bound to attribute _____ not found in 'properties' for element ________"

**Comment by Davis22xd** [comment]
same problem!

**Comment by daniele-orlando** [comment]
Same here, behaviour annotated with `/** @polymerBehavior Graphene.DataBindingBehavior */ and imported, but the linter still doesn't find the inherited properties.
